### PR TITLE
Increase horizontal padding in mobile search grid

### LINKED ISSUE #2133: Increase padding of search results grid on mobile size
## Description

The search results grid, on the mobile breakpoint, have a horizontal padding of 1em i.e. `px-4`. To be aligned with the header, they should have a horizontal padding of 1.5em i.e. `px-6`.

## Reproduction

1. Switch to a mobile breakpoint.
2. Search for something.
3. See that results grid is wider than search bar.

## Screenshots

Screenshot 2023-02-02 at 2 11 20 PM

### PR DESCRIPTION
**Pull request by dhruvkb**
## Fixes

Fixes #2133 by @dhruvkb

## Description

This PR increases the horizontal padding on the search grid on mobile to be `px-6` instead of the previous `px-4` for better alignment with the header.

## Testing Instructions

0. Check out the PR and run the dev server.
1. Search for a term.
2. See that the header and search grid are aligned on both sides.
3. Turn on the `new_header` flag.
4. Search for a term.
5. See that the header and search grid are still aligned on both sides.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like
      `Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or
      a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible
      errors.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by zackkrida** [comment]
@panchovm could you confirm this is the correct approach to take? Alternatively, I could see us decreasing the space of the header to match the existing grid.

**Comment by dhruvkb** [comment]
@zackkrida in the bug tracking sheet, @panchovm shared a [screenshot](https://share.cleanshot.com/7GT50R4N) with the text that the "purple lines should frame both sections", implying that the header is fine and the content should be narrower.

File: src/pages/search.vue [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <template>
   <VSkipToContentContainer
-    class="browse-page flex w-full flex-col px-4 md:px-10"
+    class="browse-page flex w-full flex-col md:px-10"
+    :class="[isNewHeaderEnabled ? 'px-6' : 'px-4']"
```
Comment (obulat) [inline src/pages/search.vue:4]:
```suggestion
    :class="[isNewHeaderEnabled ? 'px-6 lg:px-10' : 'px-4 md:px-10']"
```

File: src/pages/search.vue [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <template>
   <VSkipToContentContainer
-    class="browse-page flex w-full flex-col px-4 md:px-10"
+    class="browse-page flex w-full flex-col md:px-10"
```
Comment (obulat) [inline src/pages/search.vue:3]:
```suggestion
    class="browse-page flex w-full flex-col"
```

File: src/pages/search.vue (hunk 1 above)
Comment (obulat) [reply src/pages/search.vue:4]: I think that it `px-10` should start with `lg`, not `md`, to match the header

**Review by zackkrida — APPROVED** [review]
LGTM but Olga's suggestions are welcome, provided we verify they work (they seem like they would).

**Comment by panchovm** [comment]
Coming here to confirm the request and congratule the improvement. It looks great ✨

**Pull request merged**